packages/typescript/lib/quickstart/runTsc.ts

```diff
@@ -7,6 +7,53 @@ export let getLanguagePlugins: (ts: typeof import('typescript'), options: ts.Cre
 	setup?(language: Language<string>): void,
 } = () => [];
 
+export function replaceTscContent(
+	tsc: string,
+	proxyApiPath: string,
+	extraSupportedExtensions: string[],
+	extraExtensionsToRemove: string[],
+	getLanguagePluginsFile = __filename,
+) {
```

Comment:
Some JSDoc would be appreciated

Reply:
What kind of jsdoc? Description, parameters or more?

Reply:
Description of the function and of the arguments